crates/fig_util/src/process_info/linux.rs

```diff
@@ -10,6 +12,11 @@ use super::{
     PidExt,
 };
 
+pub fn terminate_process(pid: Pid) -> Result<(), String> {
```

Comment:
I duplicated this code for Unix and macOS since there are individual files for each OS. Not in love with it but it felt more cohesive than doing the OS specific logic in `mod.rs`.